guides/source/command_line.md

```diff
@@ -92,22 +92,28 @@ Rails will set you up with what seems like a huge amount of stuff for such a tin
-| `--skip-active-record`  | Skip Active Record files                                    |
-| `--skip-active-storage` | Skip Active Storage files                                   |
-| `--skip-action-cable`   | Skip Action Cable files                                     |
-| `--skip-asset-pipeline` | Skip Asset Pipeline                                         |
-| `--skip-javascript`     | Skip JavaScript files                                       |
-| `--skip-turbolinks`     | Skip turbolinks gem                                         |
-| `--skip-test`           | Skip test files                                             |
-| `--skip-system-test`    | Skip system test files                                      |
-| `--skip-bootsnap`       | Skip bootsnap gem                                           |
-
+| Argument                 | Description                                                    |
+| ------------------------ | -------------------------------------------------------------- |
+| `--skip-namespace`       | Skip namespace (affects only isolated engine)                  |
+| `--skip-collision-check` | Skip collision check                                           |
+| `--skip-git`             | Skip .gitignore file                                           |
+| `--skip-keeps`           | Skip source control .keep files                                |
+| `--skip-action-mailer`   | Skip Action Mailer files                                       |
+| `--skip-action-mailbox`  | Skip Action Mailbox gem                                        |
+| `--skip-action-text`     | Skip Action Text gem                                           |
+| `--skip-active-record`   | Skip Active Record files                                       |
+| `--skip-active-job`      | Skip Active Job                                                |
+| `--skip-active-storage`  | Skip Active Storage files                                      |
+| `--skip-action-cable`    | Skip Action Cable files                                        |
+| `--skip-asset-pipeline`  | Skip Asset Pipeline                                            |
+| `--skip-javascript`      | Skip JavaScript files                                          |
+| `--skip-hotwire`         | Skip Hotwire integration                                       |
+| `--skip-jbuilder`        | Skip jbuilder gem                                              |
+| `--skip-test`            | Skip test files                                                |
+| `--skip-system-test`     | Skip system test files                                         |
+| `--skip-bootsnap`        | Skip bootsnap gem                                              |
+| `--skip-bundle`          | Don't run bundle install                                       |
+| `--no-rc`                | Skip loading of extra configuration options from .railsrc file |
```

Comment:
```suggestion
```

And with the removal of `--no-rc`, we can keep the markdown table the same size, which makes the diff a little easier to read.